src/jest-setup.ts

```diff
@@ -1,27 +1,21 @@
-import 'isomorphic-fetch';
-import { reset } from './mockedEndpointsService';
-import { allowRealNetwork } from './settings';
-import { overrideFetch } from './overrideFetch';
-
-// make sure that axios is a singleton
-let actualAxios: any;
-jest.doMock('axios', () => {
-  if (!actualAxios) {
-    actualAxios = jest.requireActual('axios');
-  }
-  return actualAxios;
-}, { virtual: true });
+require('axios').defaults.adapter = require('./axios-fetch-adapter').default;
```

Comment:
I don't want to ask why this is outside of the beforeEach :)

Reply:
lol, yea...I have now clue sadly. it just doesn't work from within, I think it happens too late when it's in the beforeEach.